### PR TITLE
feat:add history

### PR DESCRIPTION
**Pull request by rahulmistry751**
Added history functionality

### CODE REVIEW

**Review by Pratik1005 — APPROVED** [review]
Functionality is working properly, good to merge

**Review by Mandhularajitha — APPROVED** [review]
Code looks clean
Good to Go

File: src/services/likeServices.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        console.error(error)
+    }
+}
+    // Removes liked video from liked list
+const removeLikedVideos=async (video,token,videoListDispatch)=>{
+      try {
+          const {data} = await axios.delete(`/api/user/likes/${video._id}`, {
+            headers: { authorization: token },
+          });
+          console.log(data);
```
Comment (AnjaliDeshwani) [inline src/services/likeServices.js:23]: Remove the logs if not needed

**Review by AnjaliDeshwani — APPROVED** [review]
- Clean and easy to read code
- Functionality is also working fine

**Pull request merged**